source/includes/api/_areas_v2.md

```diff
@@ -0,0 +1,280 @@
+# Areas v2
+
+The following endpoints merge together **areas of interest** and **subscriptions**. This means they can be used to transition away from subscriptions without the users loosing the subscriptions that they have already created.
```

Comment:
https://www.quora.com/What-is-the-difference-between-loosing-and-losing

Reply:
:man_facepalming: